### ISSUE TITLE
[FEATURE] show all files in a folder/subfolder

### ISSUE THREAD

**Issue by pcause** [pbek/QOwnNotes#2960]
I have directories with mixed content.  Some files are editable by qownnotes but their might be a PDF or zip or other file in the folder.  I want to see all the content in the folder displayed iand want to be able to open those files in an external app.

#### Describe the solution you'd like
Add an option to show all files, not just the ones matching the extensions listed in Settings/General.  Files that do not match the list of types that can be opened in qownnotes would be shown in a different color.  If a file that is not of a type that can be handled by qownnotes is selected, no preview is shown but the right panel has "Double click to open in default application".  There is also a right click menu item in the left pane to open in native app.

#### Describe alternatives you've considered
Didn't see an integrated solution available.  None of the plugins seemed to do this.

**Comment by pbek** [comment]
I don't think that will ever happen, this would cause so many regressions... but who knows...
I can remember a fork where someone tried something similar, but I can't remember who did it.

**Comment by pcause** [comment]
interesting.  it would seem to be a small change because you already scan directory for files making specified extensions.  So now you'd get them all.  The on click on an item in the folder pane, you check to see if a app handled file type, and if not just system open (xdg-open on linux, for example).  But your app and you know the code.

**Comment by pbek** [comment]
No, it's not a small change. 😆
